components/org.wso2.carbon.identity.organization.management.service/src/main/java/org/wso2/carbon/identity/organization/management/service/util/Utils.java

```diff
@@ -600,14 +602,62 @@ public static String getNewOrganizationVersion() {
      */
     public static boolean isLoginAndRegistrationConfigInheritanceEnabled(String tenantDomain) {
 
+        return isOrgVersionApplicable(tenantDomain,
+                OrganizationManagementConstants.OrganizationVersion.ORG_VERSION_V1);
+    }
+
+    /**
+     * Check whether claim and OIDC claim inheritance is enabled for the organization identified by the given tenant
+     * domain.
+     *
+     * Claim and OIDC claim inheritance is considered to be enabled if the organization version is
+     * equal to or greater than v1.0.0
+     *
+     * @param tenantDomain Tenant domain of the organization.
+     * @return true if the version is v1.0.0 or above, false otherwise.
```

Comment:
```suggestion
     * @return true if claim and OIDC scope inheritance is enabled, false otherwise.
```
Let's mention the abstract condition in the return statement and keep the versioning details in the above description.